spec/rails_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 require File.expand_path("../../config/environment", __FILE__)
 
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.